packages/cli/test/integration/eventbus/support/certificate.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This certificate will also expire on `Dec 16 13:24:01 2026 GMT`, which will make the tests suddenly fail. Shall we extend the validity to much longer?